cli/cmd/cloud.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/keyval-dev/odigos/cli/cmd/resources"
+	"github.com/keyval-dev/odigos/cli/pkg/kube"
+	"github.com/spf13/cobra"
+)
+
+// cloudCmd represents the cloud command
+var cloudCmd = &cobra.Command{
+	Use:   "cloud",
+	Short: "Manage odigos cloud",
+	Long:  `Used to interact with odigos managed service.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client, err := kube.CreateClient(cmd)
+		if err != nil {
+			kube.PrintClientErrorAndExit(err)
+		}
+		ctx := cmd.Context()
+
+		ns, err := resources.GetOdigosNamespace(client, ctx)
+		if resources.IsErrNoOdigosNamespaceFound(err) {
+			fmt.Println("\033[31mERROR\033[0m no odigos installation found in the current cluster. use \"odigos install\" to install odigos in the cluster or check that kubeconfig is pointing to the correct cluster.")
+			os.Exit(1)
+		} else if err != nil {
+			fmt.Printf("\033[31mERROR\033[0m Failed to check if Odigos is already installed: %s\n", err)
+			os.Exit(1)
+		}
+
+		isOdigosCloud, err := resources.IsOdigosCloud(ctx, client, ns)
+		if err != nil {
+			fmt.Println("Odigos upgrade failed - unable to read the current Odigos cloud configuration.")
```

Comment:
What's the purpose of running `odigos cloud` without any subcommand?
Is this the right error message? are we doing any upgrades here?